src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,64 @@
+package controller;
+
+import domain.*;
+import parser.BonusNumberParser;
+import parser.InputPriceParser;
+import parser.LastLottoNumbersParser;
+import view.InputView;
+import view.ResultView;
+
+import java.util.List;
+import java.util.Map;
+
+public class LottoController {
+
+    public void startLotto() {
+        int inputPrice = getInputPrice();
+        int purchaseNum = PurchaseNumberCalculator.calculate(inputPrice);
+        ResultView.printPurchaseInfo(purchaseNum);
+        List<LottoDto> lottoDtoGroup = LottoGenerator.generateLottos(purchaseNum);
+        ResultView.printLottoNumbers(lottoDtoGroup);
```

Comment:
개인적인 의견이지만 Group이라는 단어도 어떤 의미로 쓰일 수 있기 때문에 저는 리스트나 복수면 s를 붙이는 것을 선호합니다. ㅎ
```suggestion
        List<LottoDto> lottoDtos = LottoGenerator.generateLottos(purchaseNum);
        List<LottoDto> lottosDto = LottoGenerator.generateLottos(purchaseNum);
```

Reply:
확실히 변수명에 group이 들어가는 것보다 s를 붙이는게 나은 것 같아요. 수정해보겠습니다.